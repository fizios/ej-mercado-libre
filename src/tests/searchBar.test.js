import React from "react";
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import SearchBar from '../components/searchBar';

const element = (
  <MemoryRouter>
    <SearchBar />
  </MemoryRouter>
);

describe('[ SEARCH BAR ]', () => {
  it('Renders and it matches snapshot', () => {
    const wrapper = renderer.create(element);
    expect(wrapper).toMatchSnapshot();
  });
});
