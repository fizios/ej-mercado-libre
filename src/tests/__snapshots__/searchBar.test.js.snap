// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ SEARCH BAR ] Renders and it matches snapshot 1`] = `
<div
  className="header"
>
  <div
    className="search-bar"
  >
    <div
      className="logo"
    >
      <img
        alt="Mercado Libre"
        src="/logo.png"
      />
    </div>
    <div
      className="search-input"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          onChange={[Function]}
          placeholder="Nunca dejes de buscar"
          value=""
        />
        <button
          type="submit"
        >
          <img
            alt="Buscar"
            src="/icons/search.png"
          />
        </button>
      </form>
    </div>
  </div>
</div>
`;
